sdk/keyvault/Azure.Security.KeyVault.Secrets.Extensions.Configuration/api/Azure.Security.KeyVault.Secrets.Extensions.Configuration.netstandard2.0.cs

```diff
@@ -0,0 +1,34 @@
+namespace Azure.Security.KeyVault.Secrets.Extensions.Configuration
+{
+    public partial class AzureKeyVaultConfigurationOptions
+    {
+        public AzureKeyVaultConfigurationOptions() { }
+        public AzureKeyVaultConfigurationOptions(System.Uri vault, Azure.Core.TokenCredential credential) { }
```

Comment:
We use `vaultUri` in Key Vault, but if this is convention in similar use patterns, so be it.